build.sbt

```diff
@@ -38,3 +38,19 @@ lazy val core = project
       %("joda-time")
     )
   )
+  .dependsOn(metrics)
+  .aggregate(metrics)
+
+lazy val metrics = project
+  .in(file("metrics"))
+  .settings(moduleName := "frees-metrics")
+  .settings(scalaVersion := "2.12.3")
```

Comment:
You shouldn't need this setting

Reply:
I just removed the `scalaVersion`. Should I removed another settings more ?